x-pack/winlogbeat/module/security/config/winlogbeat-security.js

```diff
@@ -19,6 +19,22 @@ var security = (function () {
         "11": "CachedInteractive",
     };
 
+    var eventActionTypes = {
```

Comment:
@webmat This is adding these `event.action` values based on the event ID. Do you want these action values or would you like to hold off until the action values are more well defined?

Reply:
@andrewkroh @webmat I was thinking about  that point this morning. Personally i think is is better to define the event.action as defined in for auditbeat (some of them can be maybe reused). For example: event.action -> deleted-group-account-from
what do you thing? I can modify the event.action in order to match or follow  the same criteria as auditbeat

Reply:
I took the events actions from auditbeat'a logs and I think these event.action are more suitable.

        |"4624": "Logon Success"|"logged-in" |
        |"4625": "Logon Failed"|"logon-failed"|
        |"4634": "Logoff"|"logged-out"|
        |"4672": "Special Logon"|"logged-in-special"|
        |"4720": "Account Created"|"added-user-account"|
        |"4722": "Account Enabled"|"enabled-user-account"|
        |"4723": "Password Change"|"changed-password"|
        |"4724": "Password Reset"|"reseted-password"|
        |"4725": "Account Disabled"|"disabled-user-account"|
        |"4726": "Account Deleted"|"deleted-user-account"|
        |"4738": "Account Changed"|"modified-user-account"|
        |"4740": "Account Locked Out"|"locked-out-user-account"|
        |"4767": "Account Unlocked"|"unlocked-user-account"|
        |"4781": "Account Renamed"|"renamed-user-account"|

@andrewkroh @webmat  What do you think?

Reply:
The field `event.action` is not reserved, and can therefore be defined each source. So no limitations coming from ECS in that regard :-)

These values for `event.action` look fine to me 👍

Reply:
@andrewkroh do I change the event.actions to the ones in the table above or should I keep the previous ones?
I have already write the code to support the events for group management (**)  and some enhancements in order to  map the UAC values to a human readable form for events 4720 y 4738. If you prefer we can continue with this PR and once closed, I can create a new one with the new event.actions, new events support  and the UAC enhancements.
What do you think?

(*)
4727 (S): A security-enabled global group was created. 
4728 (S): A member was added to a security-enabled global group
4729 (S): A member was removed from a security-enabled global group
4730 (S): A security-enabled global group was deleted.
4731 (S): A security-enabled local group was created.
4732 (S): A member was added to a security-enabled local group.
4733 (S): A member was removed from a security-enabled local group.
4734 (S): A security-enabled local group was deleted.
4735 (S): A security-enabled local group was changed.
4737 (S): A security-enabled global group was changed.
4754 (S): A security-enabled universal group was created. 
4755 (S): A security-enabled universal group was changed. 
4756 (S): A member was added to a security-enabled universal group. 
4757 (S): A member was removed from a security-enabled universal group
4758 (S): A security-enabled universal group was deleted. 
4764 (S): A group’s type was changed.
4798 (S): A user's local group membership was enumerated.
4799 (S): A security-enabled local group membership was enumerated.

Reply:
I like the proposed ones that match up with auditbeat. Can you please change them in this PR? Then I'll push an update with the golden files.

Reply:
Done!